README.md

```diff
@@ -12,12 +12,12 @@ You will use Trello to report your progress on the MVP and any extra features yo
 - Don't overplan, start coding as soon as you can and adjust accordingly.  Trello is there as a guide not as a distraction
```

Comment:
Feel free to make your own Checklist.md for a cleaner README without having to dig through all this 👍